build-tools/automation/azure-pipelines.yaml

```
@@ -1115,6 +1115,67 @@ stages:
 
     - template: yaml-templates/fail-on-issue.yaml
 
+- stage: dotnet_installers
+  displayName: .NET 6
+  dependsOn: mac_build
+  jobs:
+  # Check - "Xamarin.Android (.NET 6 Preview Installers)"
+  - job: dotnet_preview_installers
+    displayName: Preview Installers
+    pool: $(HostedWinVS2019)
+    workspace:
+      clean: all
+    steps:
+    - checkout: self
+      submodules: recursive
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifactName: $(NuGetArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)\nuget-artifacts
```

You can download this directly to the expected path and skip the copy step below, the directories will automatically be created by this download if they don't exist: 
```suggestion
        downloadPath: $(System.DefaultWorkingDirectory)\bin\Build$(XA.Build.Configuration)\$(NuGetArtifactName)
```